subsys/net/lib/aws_iot/Kconfig

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig AWS_IOT
+	bool "AWS IoT library"
+	select CLOUD_API
+	select MQTT_LIB
+	select MQTT_LIB_TLS
+	select AT_CMD
+
+if AWS_IOT
+
+config AWS_IOT_STATIC_IPV4
+	bool "Enable use of static IPv4"
+
+config AWS_IOT_STATIC_IPV4_ADDR
+	string "Static IPv4 address"
+	depends on AWS_IOT_STATIC_IPV4
+
+config AWS_IOT_SEC_TAG
+	int "Security tag to use for AWS IoT connection"
+	default 42
+
+config AWS_IOT_HOST_NAME
+	string "AWS IoT server hostname"
+	default "a34x44yyrk96tg-ats.iot.eu-central-1.amazonaws.com"
+
+config AWS_IOT_PORT
+	int "AWS server port"
+	default 8883
+
+config AWS_IOT_MQTT_RX_TX_BUFFER_LEN
+	int "Buffer sizes for the MQTT library."
+	default 2048
+	help
+	  Specifies maximum message size can be transmitted/received through
+	  MQTT (exluding MQTT PUBLISH payload).
+
+config AWS_IOT_MQTT_PAYLOAD_BUFFER_LEN
+	int "Size of the buffer for MQTT PUBLISH payload."
+	default 2048
```

Comment:
I think `AWS_IOT_MQTT_RX_TX_BUFFER_LEN + AWS_IOT_MQTT_PAYLOAD_BUFFER_LEN` cannot exceed the limitations in the modem of ~2.3k. That means, the Kconfigs can exceed it, but messages that big will never be received.

Reply:
Yes, that makes sense. Those sizes were set just to make sure the size limit in those buffers would cause problems during development. I will set some reasonable stock values here, 256 for example.

Reply:
Yes, that makes sense. Those sizes were set just to make sure the size limit in those buffers would cause problems during development. I will set some reasonable stock values here, 256 for example.